nrclient/proxy_windows.go

```diff
@@ -0,0 +1,15 @@
+package nrclient
+
+import (
+	"crypto/x509"
+	"log"
+)
+
+// since Go 1.8, Go can't properly load the System root certificates on windows.
+// For more info, search Golang issues 16736 and 18609
+func systemCertPool() *x509.CertPool {
+	log.Printf("[WARNING] The Windows Infrastructure agent can't load the system root certificates. If you have set up the" +
```

Comment:
Maybe just delete "The Windows Infrastructure agent"

Reply:
Ah! I forgot to re-enable it after the refactor. Indeed, this needs to be reenabled in this PR. I'll fix it, thanks.

Reply:
Fixed!